### PR TITLE
Add https to geo admin links

### PR DESCRIPTION
**Pull request by ponceta**
This enable to run tests against https and keeps https valid for geo admin wms

#2238 (HTTPS)
#2318 (WMS.geo.admin.ch)

https://mf-geoadmin3.dev.bgdi.ch/ltpoa/?X=198293.05&Y=614529.59&zoom=3&lang=fr&topic=ech&bgLayer=voidLayer&layers=WMS%7C%7CArri%C3%A8re-plan%20hydrologique%7C%7Chttps:%2F%2Fwms.geo.admin.ch%2F%7C%7Cch.bafu.hydrologie-hintergrundkarte,WMS%7C%7CCapacit%C3%A9%20de%20r%C3%A9tention%20hydrique%7C%7Chttps:%2F%2Fwms.geo.admin.ch%2F%7C%7Cch.blw.bodeneignung-wasserspeichervermoegen

### CODE REVIEW

**Comment by ponceta** [comment]
jenkins test

File: apache/app.mako-dot-conf [hunk 1]
```diff
@@ -35,7 +35,7 @@ RewriteRule ^${apache_base_path}/[0-9]+/(img|lib|style|locales)(.*) ${apache_bas
 </LocationMatch>
 
 # Sitemaps (we are using version here to get a cached version from backend)
-RewriteRule ^${apache_base_path}/sitemap_(.*)\.xml http:${api_url}/${version}/sitemap?content=$1 [P]
+RewriteRule ^${apache_base_path}/sitemap_(.*)\.xml https:${api_url}/${version}/sitemap?content=$1 [P]
```
Comment (gjn) [inline apache/app.mako-dot-conf]: The sitemaps are not working with the given test link.

Comment (ponceta) [inline apache/app.mako-dot-conf]: I do not understand why...

https://mf-chsdi3.dev.bgdi.ch/ltpoa/1430399263/sitemap?content=base exists...

Maybe some apache trick not liking going from http to https.

**Comment by cedricmoullet** [comment]
@ponceta is it ready to merge ?

**Comment by ponceta** [comment]
Yes

**Pull request merged**